management/src/main/resources/application.properties

```diff
@@ -2,10 +2,4 @@ spring.data.mongodb.uri=mongodb://root:example@mongo:27017/admin
 #spring.data.mongodb.uri=mongodb://localhost:27017/admin
 management.publisher.topic=managementEvents
 kafka.bootstrap.address=broker:29092
-sagaEnabled=1
-metrics.publisher.api=http://metrics/metrics
-userms.hostname=http://user
-userms.port=8080
-#userms.hostname=http://localhost
-#userms.port=8083
-#server.port=8081
```

Comment:
Should we be removing these entries? How do we default to config entries in case no ENV variable specified?